src/jit/emit.h

```diff
@@ -932,7 +932,7 @@ class   emitter
             }
             void            iiaSetInstrCount(int count)
             {
-                assert(abs(count < 10));
+                assert(count < 10);
```

Comment:
Further investigation shows that this should be assert(abs(count) < 10). A negative value for count can occur, at least comments indicate such, it is called from https://github.com/dotnet/coreclr/blob/master/src/jit/emitarm.cpp#L4283 where as per the comment, a negative count indicates it should jump back count instructions.